src/coprocessor/codec/mysql/json/mod.rs

```diff
@@ -47,3 +53,77 @@ pub enum Json {
     Boolean(bool),
     None,
 }
+
+// https://dev.mysql.com/doc/refman/5.7/en/json-creation-functions.html#function_json-array
+pub fn json_array(elems: Vec<Datum>) -> Result<Json> {
```

Comment:
Wooo, I rebase my branch with master, and there are some commits that have been merged into master but haven't been merged in pre-ga